smoketest/harness/graphql.go

```diff
@@ -63,6 +64,20 @@ func (h *Harness) SetConfigValue(id, value string) {
 	h.Trigger()
 }
 
+// SetSystemLimit will update the value of a system limit given an id (e.g. `RulesPerSchedule`).
+func (h *Harness) SetSystemLimit(id limit.ID, value int) {
```

Comment:
I'm not sure if a func to directly manipulate the DB is appropriate in the harness' `graphql.go` file, especially if it's not in the current schema

Reply:
The test harness should be okay for this, for now. All the limits are implemented as DB triggers, so we don't need to sync with the engine or anything and will work as intended. The harness accesses the DB directly for things like time-shifting, so it's not a new connection.

But it is intended to be temporary until we build a new mutation & query. We decided not to add any new GraphQL mutations in this scope, and instead leave that for when we add limits config it to the Admin page so we can account for any "hidden" requirements. At that time we will update this to use the mutation instead of direct-to-DB.

However, it might be worth adding a comment inside the method to indicate that the query is only temporary, and should be replaced with a GQL call once system limits have been added to the schema. 

@dctalbot would you mind adding that?

Reply:
Sounds good, yeah a `todo` that makes this known would be preferable.